### DISCUSSION TITLE
Diffuse irradiance perceived by the ground ( ground diffuse acceptance)

### DISCUSSION THREAD

**Discussion by 115522** [pvlib/pvlib-python#1807] [Q&A]
hello pvlib community ?

I want modelize bifacial system and calculate the ground diffuse acceptance , 

![image](https://github.com/pvlib/pvlib-python/assets/32468472/7571984e-c75c-49e2-b2fb-d5f4794ae2ae)

is this modelized by _vf_ground_sky_integ  function on infinite_sheds  

thanks in advance

**Comment by cwhanse** [comment]
The functions have changed in v0.10.0 (moved to pvlib.bifacial.utils and made public). In versions prior to v0.10.0, the functions are in pvlib.bifacial.infinite_sheds and are private (start with "_").

I think you want either `pvlib.bifacial.utils.vf_ground_sky_2d` (fraction of sky dome visible at a point) or `pvlib.bifacial.utils.vf_ground_sky_2d_integ` (averaged over a range of points).
